pkg/ui/changelog/changelog.go

```diff
@@ -0,0 +1,58 @@
+package changelog
+
+import (
+	"errors"
+
+	tea "github.com/charmbracelet/bubbletea"
+	changelog "github.com/futurice/jalapeno/pkg/ui/changelog/prompt"
+)
+
+func RunChangelog() ([]string, error) {
+	var changelog []string
+	verInc, err := RunSelectPrompt()
+
+	if err != nil {
+		return []string{}, errors.New("failed to get version type")
+	}
+
+	logmsg, err := RunTextAreaPrompt()
+
+	if err != nil {
+		return []string{}, errors.New("failed to get log message")
+	}
+
+	changelog = append(changelog, verInc)
+	changelog = append(changelog, logmsg)
+
+	return changelog, nil
+}
+
+func RunSelectPrompt() (string, error) {
```

Comment:
Since `RunSelectPrompt()` and `RunTextAreaPrompt()` are never used only by themselves, they should be private functions. So they should be renamed `runSelectPrompt()` and `runTextAreaPrompt()` respectively.